### PR TITLE
💄 SolutionExplorer Layout Fix

### LINKED ISSUE #550: 💄 Inconsitent Layout of SolutionExplorer
#### Describe your issue

grafik

I see the following issues:

1. The ✖️ Icon is not placed the same way
2. The ✖️ Icon does not have any tooltip
3. `Opened Diagrams` should be `Solution Explorer`
4. Below `Solution Explorer` add another `<hr>` and then the content box for `Opened Diagrams`
5. The _Open a new Solution Icon_ and the Icon in the solution list is the same, the _Open a new Solution Icon_ should have a + somewhere, just like the _Open Diagram Icon_
6. There is no description on these icons 
grafik

#### How can others reproduce the issue?

Look at the Solution Explorer

#### Possible solution

1. Layout the ✖️ Icons the same way
2. Add a `title` attribute
3. Rename `Opened Diagrams` to `Solution Explorer`
4. Add a new caption above the opened diagrams: `Opened Diagrams`
5. Add a folder icon with a plus (maybe use stacked icons)
6. Add atitle attribute

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### LINKED ISSUE #551: 🐛 Wrong Hover Event in Solution Explorer
#### Describe your issue

hover-bug

#### How can others reproduce the issue?

See the gif

#### Possible Solution

Fix the hover effect

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by Vyperus**
## What did you change?

Closes #551 
Closes #550

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 1]
```diff
@@ -35,7 +50,10 @@
   visibility: hidden;
 }
 
-
+.solution-input {
+  visibility: hidden;
+  height: 0;
```
Comment (alexanderkasten) [inline src/modules/process-solution-panel/process-solution-panel.scss]: move the height property to the top

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 2]
```diff
@@ -46,77 +64,91 @@
...
+.solution-diagram-group__link {
+  width: 180px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.solution-diagram-group__link:hover {
+  text-decoration: underline;
   cursor: pointer;
```
Comment (alexanderkasten) [inline src/modules/process-solution-panel/process-solution-panel.scss]: move the cursor property to the top

**Comment by SteffenKn** [comment]
@alexanderkasten done

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 3]
```diff
@@ -46,77 +64,91 @@
...
+  cursor: pointer;
+  color: #5e5e5e;
 }
 
-.index-card {
-  padding: 8px 12px;
-  color: darkgray;
+.solution-input-group {
+  text-align: center;
+  margin: 20px 0;
```
Comment (alexanderkasten) [inline src/modules/process-solution-panel/process-solution-panel.scss]: move the margin property to the top

File: src/modules/process-solution-panel/process-solution-panel.scss [hunk 4]
```diff
@@ -18,6 +18,10 @@
   color: darkgray;
```
Comment (alexanderkasten) [inline src/modules/process-solution-panel/process-solution-panel.scss]: please add `cursor: pointer;` to this css class

Comment (SteffenKn) [reply src/modules/process-solution-panel/process-solution-panel.scss]: done

**Pull request merged**